### ISSUE TITLE
Warning: error reading values after @if  

### ISSUE THREAD

**Issue by arnoldbird** [sass/libsass#1899]
I have had success with libsass and grunt in the past, but in my current project, when I run grunt watch and edit one of my scss files, I get warnings like...

Warning: node_modules/zen-grids/sass/zen-grids/grids:23: error reading values after @if
 Use --force to continue.

If I then go into the module (zen-grids, or whatever) and fix or comment out the problem, I will encounter another such warning.  There is no end to these warnings.  It isn't possible to fix everything the warnings mention, and in some cases these warning seem to be false positives.  Every fix simply leads to a new warning.  I don't know where the problem may lie, so it's difficult to research this.  I'm using Ubuntu 15.04.

My package.json looks like...

{
  "name": "my_project",
	"version": "0.0.1",
	"devDependencies": {
		"grunt": "0.4.5",
		"grunt-contrib-watch": "^0.6.1",
		"grunt-sass": "^0.16.0",
        "grunt-contrib-concat": "^0.5.1",
        "grunt-contrib-uglify": "^0.11.0",
        "zen-grids": "^2.0.1"
	}
}

If I run sassc -v, I see...
sassc: 3.2.1
libsass: 3.2.1
sass2scss: 1.0.3

**Comment by xzyfer** [comment]
Libsass 3.2.1 is old, please update.
Most importantly if you can't take the time to supply some sample Sass code to produce the error, we can't possible begin to debug the problem. We will not take the time to debug an entire framework to find a bug that may not exist.

**Comment by arnoldbird** [comment]
Thanks.  I know when I post a report like this, there is a chance I'm posting to the wrong project, but it's very hard for me to know where the problem lies.  In any case, something wasn't playing nice with something else, but I've since found a good combination for this project, based around compass and gulp-simple-compass.  In another project, I found that libsass + grunt-sass worked much better than compass.  An upgrade to Ubuntu 15.04 may be a factor this time (pure speculation), and/or the old libsass version I was using.  

Perhaps it would make sense to close this.  : )
